rcl/test/rcl/test_arguments.cpp

```diff
@@ -317,8 +318,16 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_bad_alloc_unpar
   EXPECT_EQ(
     RCL_RET_BAD_ALLOC, rcl_arguments_get_unparsed(&parsed_args, bad_alloc, &actual_unparsed));
   rcl_reset_error();
+
   EXPECT_EQ(
-    RCL_RET_BAD_ALLOC, rcl_arguments_get_unparsed_ros(&parsed_args, bad_alloc, &actual_unparsed));
```

Comment:
@Blast545 qq: why remove this test?

Reply:
Don't remember erasing it, restored it af3adb5